Jenkinsfile

```diff
@@ -158,6 +158,23 @@ pipeline {
             echo 'Skipping Kafka Streams archetype test for Java 15'
           }
         }
+
+        stage('ARM') {
+          agent { label 'arm4' }
+          options {
+            timeout(time: 8, unit: 'HOURS') 
+            timestamps()
+          }
+          environment {
+            SCALA_VERSION=2.12
+          }
+          steps {
```

Comment:
Would it be better to add error processing, for example "catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE')" .
In this case, the arm CI fails, which does not affect the CI build result of the entire Kafka.